### ISSUE TITLE
Pagination plugin's page size select provides no context for screen readers

### ISSUE THREAD

**Issue by bobbylight** [SitePen/dgrid#852]
Screen readers such as JAWS don't give a good description of what the page size select from the Pagination plugin is used for.  When tabbing to it, it will read something like this:

"Table Combo box, 25. To change the selection, use the arrow keys"

There should be accessible text set on the select stating something along the lines of "Number of visible rows" to make JAWS give a better description of the input.
